atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfigs.java

```diff
@@ -37,21 +37,18 @@
 import com.palantir.config.crypto.jackson.JsonNodeVisitors;
 import com.palantir.remoting.api.config.ssl.SslConfiguration;
 
-import io.dropwizard.jackson.DiscoverableSubtypeResolver;
-
 public final class AtlasDbConfigs {
```

Comment:
note: this class is effectively not used internally, so instead of reimplementing we might just want to cut our losses and just remove.

Reply:
would let this become a cool -270 lines.